tools/cythonize.py

```diff
@@ -71,15 +71,28 @@ def process_pyx(fromfile, tofile, cwd):
             for line in pt:
                 if "cython" not in line.lower():
                     continue
-                _, line = line.split('=')
-                required_version, _ = line.split('"')
+                line = ''.join(line.split('=')[1:])  # get rid of "Cython>="
```

Comment:
perhaps worth noting that the control flow here is somewhat fragile to comments in `pyproject.toml` that contain the string `Cython`; I just decided to purge out my comments in that file rather than trying make this a more sophisticated parser (which seemed like a bad use of time).

Reply:
good call